### PR TITLE
Added stl files for plates with support of different combos

### PR DESCRIPTION
**Pull request by Sushi6006**
Added a new directory containing stl files for different combos of rotary encoders and switches, as well as a readme. Supports dumbpad v1.x and fits in Anthony @ customkbd.com.au's case. Fully tested.

### CODE REVIEW

**Comment by imchipwood** [comment]
I'll try to print one of these this week to see if they work with my version of Anthony's case (don't see why they wouldn't, I only added small supports in the corners below the PCB).

Will get back to you soon!

**Comment by Sushi6006** [comment]
Great! Looking forward to your response.

**Comment by imchipwood** [comment]
Sorry for the delay!

I printed out the full 4x4 plate and it seems to fit as expected.

However, I think you only need two STL files to handle an encoder in any of the four corners of the 4x4 grid. Let's take "top_left" and "top_right", for example, and then run through each case where a single encoder is in one of the 4x4 corners:
1. encoder in top left: obviously "top_left" design works
2. encoder in top right: again, "top_right" is the obvious choice
3. encoder in bottom left: flip "top_left" design along the X axis before printing
4. encoder in bottom left: flip "top_right" design along X axis before printing

Similarly, only one of the "two encoders" STL files is needed.

Remove the excess files and I'll approve the PR.

**Comment by imchipwood** [comment]
You might want to include the info about how to rotate the plate to support the various encoder positions in the readme.

Finally, it wasn't immediately obvious how to orient the plate with the switches. It appears that the thin edge needs to be near the pro micro? Mention that in the readme too!

And remove the stuff about "my first time 3d printing" from the readme. It should only be about the design, not you :)

edit: I just noticed I talk about myself in the case readme, so maybe I'm being hypocritical here... hah

**Comment by Sushi6006** [comment]
I thought about that. However, I figured people may have ideas for customising top surface while printing (i.e. change the filament colour), and that is the reason why I kept all four single-encoder stl files. I will be more than happy to remove two of them and add a description if that's what you prefer though. Let me know what you think and I will do whatever you says :)

File: case/plate/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## 3D-Printable Mounting Plate for dumbpad
+Mounting plates designed for dumbpad v1.x ([dumbpad by imchipwood](https://github.com/imchipwood/dumbpad/)); fits in the case designed by Anthony @ [customkbd.com.au](https://customkbd.com.au/); 1.5mm thick.
+
+I found out that the hot-swap sockets I soldered in cannot hold switches firmly enough, and plates were needed. Hence, I started this project.
+
+I am new to 3D printing and 3D modelling, and this is my first ever model. Please be forgiving and feel free to point out any inappropriates in my design. PR's are welcomed too.
```
Comment (imchipwood) [inline case/plate/README.md]: Remove this line and we're good to go!

**Comment by imchipwood** [comment]
Hey sorry about the delay getting back to you on this. 

Just remove that line and I'll approve the PR! I must've missed the note in the readme about how to orient the switch plate - though I didn't seem to have any issue fitting the plate in my version of Anthony's case no matter which way I oriented it.

**Comment by Sushi6006** [comment]
That is totally fine. I have removed the line you mentioned - sorry about my unprofessional wordings. Please check again and see if the PR can be merged.

**Review by imchipwood — APPROVED** [review]
No worries! Thanks for making the switch plates!

**Pull request merged**